README.rst

```diff
@@ -169,6 +169,23 @@ Events
     # Get latest stream position
     client.events().get_latest_stream_position()
 
+Metadata
+~~~~~~~~
+
+.. code-block:: python
+
+    # Get metadata
+    client.file(file_id='SOME_FILE_ID').metadata().get()
+
+    # Create metadata
+    client.file(file_id='SOME_FILE_ID').metadata().create({'key', 'value')
+
+    # Update metadata
+    from boxsdk.object.metadata import MetadataUpdate
```

Comment:
Should we start putting classes for external access to the __init__ .py of the boxsdk package? Then users can do `from boxsdk import MetadataUpdate`.